data/translations/tl/accordion.json

```diff
@@ -0,0 +1,32 @@
+{
+    "header": "Ang Vote.gov ay Tumutulong sa Inyo:",
+
+    "title_1": "Magparehistro para Bumoto",
+    "content_1_part_1": "Ang lahat ng mga estado maliban sa North Dakota ay kinakailangan na kayo ay magparehistro bago bumoto sa isang eleksyon. Piliin ang inyong estado o teritoryo mula sa dropdown na menu para mahanap ang mga panuntunang nararapat sa inyo.",
+    "content_1_part_2": "Ang mga mamamayan ng U.S. na naninirahan sa labas ng U.S., kabilang ang mga sundalo ng U.S. at kanilang mga pamilya, ay maaaring magparehistro para bumoto at humiling ng absentee ballot sa pamamagitan ng pagkumpleto ng Federal Post Card Application %link%. Ang mga miyembro ng pamilyang militar ng U.S., tulad ng iba, ay dapat matugunan ang mga kinakailangan sa pagiging karapat-dapat na botante ng U.S. para magparehistro at humiling ng isang balota. Para sa karagdagang impormasyon, bisitahin ang Programa ng Tulong sa Pederal na Botante.",
+    "content_1_part_2_link": "(FPCA Aplikasyon ng Pederal Post Kard)",
```

Comment:
Remove the wrapping parentheses from this string.

Reply:
fixed